apps/picsa-tools/option-tool/src/app/pages/home/home.component.scss

```diff
@@ -144,3 +144,7 @@ th.split-values {
   justify-content: center;
   gap: 10px;
 }
+
+.button-margin {
+  margin-left: 1300px;
```

Comment:
nit(blocking) - putting a fixed margin won't work on smaller screens. Normally the solution would be to specify `margin-left: auto` which will use up as much space as possible, however this doesn't work on `mat-button` elements because they're designed to be displayed inline, `display: inline-flex`.

So in order to fix this you should set both `display: block; margin-left:auto` and then to ensure it overwrites the default button styles probably better to put it directly on the html component itself instead of css class, i.e.
```html
<button style="display: block; margin-left: auto"
```
or use a more specific css selector, e.g.
```css
button.button-margin{
    display: block; 
    margin-left: auto
}
```

Reply:
thank you for the tip